apps/cowswap-frontend/src/modules/ordersTable/containers/OrdersTableWidget/index.tsx

```diff
@@ -148,8 +179,11 @@ export function OrdersTableWidget({
 
   useValidatePageUrlParams(orders.length, currentTabId, currentPageNumber)
 
+  const ordersToCheckPendingPermit = useGetOrdersToCheckPendingPermit(ordersList, chainId, balancesAndAllowances)
+
   return (
     <>
+      <PendingPermitUpdater orders={ordersToCheckPendingPermit} />
```

Comment:
I'm not sure that `permits` module should depends on `ordersTable`.
Maybe we should move the updater to `App/Updaters.ts`?